change branch name and file names

xumegan opened the pull request:
test will need to install dependence of styled-components

gvanbo committed:
some progress on metronome function

gvanbo committed:
moved new audio inside of useEffect

gvanbo committed:
moved audio inside of useEffect

gvanbo committed:
eslint init JS modules standard formating enforced

kyo6919 committed:
Merge pull request #4 from Solid-Time-Metronome/installTesting

eslint init JS modules standard formating enforced

kyo6919 committed:
Merge branch 'main' into addTempoFeature

gvanbo committed:
routed basic function console log for tempo

gvanbo committed:
temporarily removed setTempo

gvanbo committed:
Merge branch 'addTempoFeature' of https://github.com/Solid-Time-Metronome/webapp into addTempoFeature

gvanbo committed:
installed PropTypes and updated code

gvanbo committed:
onBPMselect code is in place

gvanbo committed:
tempo component now speaks to visualize

gvanbo committed:
removed time const and updated BPM logic

gvanbo commented:
This branch is behind the main. Please sync with the main branch, and then push this branch.  (you might need to sync with main and run npm install for prop-types and eslint

gvanbo committed:
Merge pull request #6 from Solid-Time-Metronome/addTempoFeature

Add tempo feature

gvanbo committed:
removed auto react elements (fav, logo)

gvanbo committed:
fixed alll linting and proptype issues

xumegan committed:
Merge pull request #7 from Solid-Time-Metronome/fixLinting

Fix linting

Commit:
matching eslint format, change branch and file names

gvanbo committed:
onBPMSelect not setting correct value.

kyo6919 committed:
teams page + router + nav bar

-added teams component, page, and nav bar(both desktop and mobile hamburger)
- added react router dom.
-added utility hook to change document title.

-having CSS structure issues, need to talk with team to fix up.

-moved main.style.css to App.css so that it can be global css.

kyo6919 committed:
eslint issues

gvanbo committed:
Merge pull request #8 from Solid-Time-Metronome/ShowBeats

matching eslint format, change branch and file names

kyo6919 committed:
teams page updates

kyo6919 committed:
added css width

kyo6919 committed:
Merge branch 'main' into teams-page

gvanbo committed:
Merge pull request #9 from Solid-Time-Metronome/teams-page

Teams page

gvanbo committed:
Merge branch 'main' of https://github.com/Solid-Time-Metronome/webapp

Commit:
change showbeats to cycles, and sync with beats

Commit:
fixing css

gvanbo committed:
Merge pull request #10 from Solid-Time-Metronome/display-beats-in-cycle

Display beats in cycle

kyo6919 committed:
v1

section ready css

gvanbo committed:
Merge pull request #12 from Solid-Time-Metronome/css-section-ready

v1

gvanbo committed:
prep for merge with main

gvanbo committed:
Merge branch 'main' of https://github.com/Solid-Time-Metronome/webapp

gvanbo committed:
navBar simplified and completed

kyo6919 committed:
Merge pull request #13 from Solid-Time-Metronome/styleNavBar

Style nav bar

Commit:
get beats value from drop down

kyo6919 committed:
single button

gvanbo committed:
Merge pull request #17 from Solid-Time-Metronome/Single-Button

single button

gvanbo committed:
Merge pull request #16 from Solid-Time-Metronome/add-beats

get beats value from drop down

Commit:
add volume

Commit:
one volume works

Commit:
min changes

Commit:
min changes

gvanbo committed:
Merge pull request #18 from Solid-Time-Metronome/add-volume

Add volume

gvanbo committed:
Update README.md

Commit:
remove showbeats and move text up

Commit:
finished

Commit:
make background

Commit:
volume works without initial state

gvanbo committed:
Merge pull request #21 from Solid-Time-Metronome/move-volume-and-remove-showbeats

Move volume and remove showbeats

gvanbo committed:
removed import react in tempoDropdown

gvanbo committed:
initial tempo value and colors for visualize

Commit:
set initial value works, but input css wont change

Commit:
volume works

Commit:
volume works well

gvanbo committed:
oct 15 progress

kyo6919 committed:
contact page with form

gvanbo committed:
Merge pull request #22 from Solid-Time-Metronome/working-inital-volume

Working inital volume

gvanbo committed:
Merge pull request #23 from Solid-Time-Metronome/emailjs

contact page with form

Commit:
fix git api fetch format

gvanbo committed:
Merge pull request #24 from Solid-Time-Metronome/gitapifetch

fix git api fetch format
